lib/render.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-env es2018 */
+const errors = require('./errors');
+const request = require('request');
+const { version } = require('../package.json');
+const _ = require('lodash');
+const generateJwt = require('./generateJwt.js');
+
+/**
+* An object representing an OpenTok render.
+* 
+* @property {String} id
+*   The ID of the render instance
+* 
+* @property {String} projectId
+*   The ID of the project for the render.
+* 
+* @property {String} sessionId
+*   The ID of the session being rendered into.
+* 
+* @property {Number} createdAt
+*   The time at which the render was created, in milliseconds since the UNIX epoch.
+** 
+* @property {Number} upddatedAt
+*   The time at which the render was created, in milliseconds since the UNIX epoch.
+*
+* @property {String} url
+*   A publically reachable URL controlled by the customer and capable of generating 
+*   the content to be rendered without user intervention.
+* 
+* @proerpty {String} status
+*   Current status of for the render. Will be one of `started`, `stopped` or `failed`
```

Comment:
The REST docs also mention "starting". Should these be strings (e.g., `"started"`)?